envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/echo/EchoServiceExtension.kt

```diff
@@ -1,24 +1,6 @@
 package pl.allegro.tech.servicemesh.envoycontrol.config.echo
 
-import org.junit.jupiter.api.extension.AfterAllCallback
-import org.junit.jupiter.api.extension.BeforeAllCallback
-import org.junit.jupiter.api.extension.ExtensionContext
+class EchoServiceExtension : ServiceExtension(EchoContainer()) {
```

Comment:
How about generics?
```kotlin
open class  ServiceExtension<T: ServiceContainer>(val container: T) : BeforeAllCallback, AfterAllCallback {
```
```kotlin
class EchoServiceExtension : ServiceExtension<EchoContainer>(EchoContainer()) {
```

Reply:
Wow thanks! 😮   I went down a bad path trying to do something different at first and couldn't return 😲